frontend/composables/useMSAL.ts

```diff
@@ -236,7 +236,7 @@ export const useMSAL = async () => {
     const request: SilentRequest = {
       scopes,
       forceRefresh: false, // Set this to "true" to skip a cached token and go to the server to get a new token
-      account: inst.getAccount({ homeAccountId: account.value.homeAccountId }) ?? undefined,
+      account: inst.getAccountByHomeId(account.value.homeAccountId) ?? undefined,
```

Comment:
[`getAccountByHomeId` is deprecated](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/0f5e6c2142889ea9f43d808f270aec9264afd363/lib/msal-browser/src/app/PublicClientApplication.ts#L207), which is why I changed it. If you're seeing otherwise, you might not have run `npm install` after pulling in my most recent changes, which updated the MSAL library.

Reply:
Ahhh yes that's it. I was getting TSC failures.